lib/project_types/script/config/extension_points.yml

```diff
@@ -3,13 +3,16 @@ discount:
     package: "@shopify/extension-point-as-discount"
     version: "~0.0.21"
     sdk-version: "^4.0.0"
+    toolchain-version: "~0.1.0"
```

Comment:
Actually can you change the `~` back to a `^` since we're using minor versions? We just need the `~` for things that only have patch versions (which I would like us to avoid going forward).